ruby-challenges.rb

```diff
@@ -22,6 +54,16 @@
 beatles_album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def vowels_be_gone string
+    string.delete "AEIOUaeiou"
+end
+
+p "-----------This is challenge two: -----------"
+p vowels_be_gone beatles_album1 #"Rbbr Sl"
+p vowels_be_gone beatles_album2 #"Sgt Pppr"
+p vowels_be_gone beatles_album3 #"bby Rd" 
+
+# I learned a lot with this problem. The syntax wasn't as obvious as I thought and ended up spending a good amount of effort with it. 
 
```

Comment:
Glad you learned a lot from it! Sometimes it's the simple things that stump us for a while isn't it? Haha